GNUmakefile

```diff
@@ -35,21 +35,39 @@ OBJS += portable/common/reallocarray.o
 else ifeq ($(UNAME_S),NetBSD)
 GROUP =	bin
 OBJS += portable/common/reallocarray.o
+else ifeq ($(UNAME_S),Darwin)
+GROUP =	bin
+OBJS += portable/common/reallocarray.o portable/common/strtonum.o
 endif
 
 #
 # Only update /etc/shells at install time, not during update
 #
+ifneq ($(UNAME_S),Darwin)
```

Comment:
I've put lots of guards in because I didn't want to change they way you have things set up on other platforms.